src/routes/subscribe/addSubscriber.ts

```diff
@@ -0,0 +1,69 @@
+import { RequestHandler } from 'express';
+
+import responseTemplate from '../../utils/responseTemplate';
+import Subscriber from '../../models/Subscriber';
+import { validateEmail } from '../../utils/validateEmail';
+
+const addSubscriber: RequestHandler = async (req, res) => {
+    try {
+        const { email } = req.body.toLowerCase();
```

Comment:
This doesn't work. Make sure the email is received from req.body and then lowerCase it in another variable.